index.html

```diff
@@ -8,135 +8,172 @@
+          alt="bookmark"
+          class="app-grid__card--icon"
+        />
         <h2>4+4?</h2>
-        <details class="app-grid__main__card__show-answer">
-          <summary class="app-grid__main__card__show-answer--toggle-show">
+        <details class="app-grid__card__show-answer">
+          <summary class="app-grid__card__show-answer--toggle-show">
             Show Answer
           </summary>
-          <p class="app-grid__main__card__show-answer__answer">8</p>
+          <p class="app-grid__card__show-answer__answer">8</p>
 
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
         </details>
       </section>
 
-      <section class="app-grid__main__card">
+      <section class="app-grid__card">
+        <img
+          src="img/icons/bookmark-regular.svg"
+          alt="bookmark"
+          class="app-grid__card--icon"
+        />
         <h2>5+5?</h2>
-        <details class="app-grid__main__card__show-answer">
-          <summary class="app-grid__main__card__show-answer--toggle-show">
+        <details class="app-grid__card__show-answer">
+          <summary class="app-grid__card__show-answer--toggle-show">
             Show Answer
           </summary>
-          <p class="app-grid__main__card__show-answer__answer">10</p>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <p class="app-grid__card__show-answer__answer">10</p>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
         </details>
       </section>
     </main>
 
-    <footer class="app-grid__footer">
-      <nav class="app-grid__footer__nav">
+    <footer class="footer">
+      <nav class="footer__nav">
```

Comment:
Here I would give a different class name. The navigation is it's own block and doesn't have to be an element of the footer. This article is a good resource to read about BEM 🙂 : https://css-tricks.com/bem-101/
```suggestion
      <nav class="navigation">
```